cmd/bosun/expr/funcs.go

```diff
@@ -795,6 +801,25 @@ func sum(dps Series, args ...float64) (a float64) {
 	return
 }
 
+func Des(e *State, T miniprofiler.Timer, series *Results, alpha float64, beta float64) *Results {
+	for _, res := range series.Results {
+		sorted := NewSortedSeries(res.Value.Value().(Series))
+		if len(sorted) > 1 {
```

Comment:
continue here instead of putting the logic in a deeper block: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow